src/qt/walletview.cpp

```diff
@@ -197,6 +200,77 @@ void WalletView::gotoVerifyMessageTab(QString addr)
         signVerifyMessageDialog->setAddress_VM(addr);
 }
 
+void WalletView::gotoLoadPSBT()
+{
+    QString filename = GUIUtil::getOpenFileName(this,
+        tr("Load Transaction Data"), QString(),
+        tr("Partially Signed Transaction (*.psbt)"), nullptr);
+    if (filename.isEmpty()) return;
+    std::ifstream in(filename.toLocal8Bit().data(), std::ios::binary);
+    // https://stackoverflow.com/questions/116038/what-is-the-best-way-to-read-an-entire-file-into-a-stdstring-in-c
+    std::string data(std::istreambuf_iterator<char>{in}, {});
+
+    std::string error;
+    PartiallySignedTransaction psbtx;
+    if (!DecodeRawPSBT(psbtx, data, error)) {
+        Q_EMIT message(tr("Error"), tr("Unable to decode PSBT file") + "\n" + QString::fromStdString(error), CClientUIInterface::MSG_ERROR);
```

Comment:
The error message could include the file path. BTW, is the error message relevant?
<img width="532" alt="Screenshot 2020-03-15 at 10 38 30" src="https://user-images.githubusercontent.com/3534524/76699841-30a75400-66a9-11ea-980f-2eba4a8b3ee6.png">

Reply:
I think the error message is useful for debugging. This particular one shouldn't happen anymore, thanks to the size check.

I don't think adding the file name is useful, since the user selected it themselves.